rasa_core/policies/fallback.py

```diff
@@ -87,7 +88,7 @@ def predict_action_probabilities(self,
         The fallback action is predicted if the NLU confidence is low
         or no other policy has a high-confidence prediction.
         """
-
+        self.core_threshold_set = False
```

Comment:
This flag wont work: The policy (as part of the model) is shared by all the conversations. Because we are using async, there should be no state within that class that is specific to a conversation (as the server might receive a message while we are still handling one, due to asyncio we might jump into handling that other message, even though the first one is not finished yet)

Reply:
> there should be no state within that class that is specific to a conversation

@tmbo Good to know -- I'd love a clarification sometime about how async changes things, and what mistakes like this to avoid as a result

Reply:
Ok. In that case we might go with the idea of @erohmensing: evaluate the fallback policy always at last and log the message if the fallback policy was chosen and the obtained confidence is equal to the core_threshold.